master/buildbot/master.py

```diff
@@ -132,6 +133,9 @@ def create_child_services(self):
         self.metrics = metrics.MetricLogObserver()
         self.metrics.setServiceParent(self)
 
+        self.stats_service = stats_service.StatsService(self)
```

Comment:
`master.stats_service` is used in the code to call the `StatsService.post*` methods.

I think that you might be confusing the StatsService and storage backends. StatsService isn't user controlled - so, a user needn't touch the config file to have to do anything with the StatsService. Also, it runs reference to the running service instance is stored in `master`. On the other hand, the config file is needed to configure and add a storage backend, the way you mentioned in the comment. This list of storage backends is stored in the config.